Snippets.md

```diff
@@ -362,6 +362,8 @@
 ### Applied rule message
 * Remove messages on Checkout page  
   Snippet: https://gist.github.com/AnanthFlycart/d0b500bb0fd74cc20db854e817be04be  
+* Decrease showAppliedRulesMessages event priority  
```

Comment:
@kirubaFlycart Improve the message. what was the case

Reply:
Updated